includes/admin/class-wcmypa-admin.php

```diff
@@ -1655,6 +1656,46 @@ public static function canUseWebhooks(): bool
 
         return false;
     }
+
+    public function saveOrderNote(int $commentId, WC_Order $order): void
+    {
+        $orderMeta = get_post_meta($order->get_id(), self::META_PPS);
+        $lastOrder = end($orderMeta);
+        $orderUuid = $lastOrder ? $lastOrder[self::META_PPS_UUID] : null;
+
+        if (! $orderUuid) {
+            return;
+        }
+
+        $comment    = get_comment($commentId);
+        $orderNotes = (new OrderNotesCollection())->setApiKey(
+            WCMYPA()->setting_collection->getByName(
+                WCMYPA_Settings::SETTING_API_KEY
+            )
+        );
+        $orderNote  = new OrderNote([
+            'note'      => $comment->comment_content,
+            'author'    => 'webshop',
+            'orderUuid' => $orderUuid,
+        ]);
+
+        try {
```

Comment:
waarom dubbel? Dit is dan wel "oud", maar je kunt dit nog steeds in een service oid stoppen waardoor je geen code hoeft te dupliceren